offboarding steps (#1646)
fastify/fastify: CONTRIBUTING.md

2. Choose which team to join *(more than one is ok!)* based on how you want to help.
3. Open a pull request to [`fastify/fastify:master`](https://github.com/fastify/fastify/pulls) that adds your name, username, and email to the team you have choosen in the [README.md](./README.md) and [package.json](./package.json) *(if you are part of the core team)* files. The members lists are sorted alphabetically; make sure to add your name in the proper order.
4. Open a pull request to [`fastify/website:master`](https://github.com/fastify/website/pulls) adding yourself to the [team.yml](https://github.com/fastify/website/blob/master/src/website/data/team.yml) file. This list is also sorted alphabetically so make sure to add your name in the proper order. Use your GitHub profile icon for the `picture:` field.
5. The person that does the onboarding must add you to the [npm org](https://www.npmjs.com/org/fastify), so that you can help maintaining the official plugins.

### Offboarding Collaborators

We are thankful to you and we are really glad to have worked with you.
We'll be really happy to see you here again if you want to come back, but for now the person that did the onboarding must:

> 1. Ask the collaborator if he/she want to stay or not.

Reviewer: ```suggestion
1. Ask the collaborator if they want to stay or not.
```

> 1. If the collaborator can't work with us anymore, he/she should:

Reviewer: ```suggestion
1. If the collaborator can't work with us anymore, they should:
```

>   1. Open a pull request to [`fastify/fastify:master`](https://github.com/fastify/fastify/pulls) and move himself/herseff in the *Past Collaborators* section.

Reviewer: ```suggestion
  1. Open a pull request to [`fastify/fastify:master`](https://github.com/fastify/fastify/pulls) and move themselves in the *Past Collaborators* section.
```

>   2. Open a pull request to [`fastify/website:master`](https://github.com/fastify/website/pulls) and move himself/herseff in the *Past Collaborators* section in the [team.yml](https://github.com/fastify/website/blob/master/src/website/data/team.yml) file.

Reviewer: ```suggestion
  2. Open a pull request to [`fastify/website:master`](https://github.com/fastify/website/pulls) and move themselves in the *Past Collaborators* section in the [team.yml](https://github.com/fastify/website/blob/master/src/website/data/team.yml) file.
```
